markdown/chapters/meandeck/alternative-piles.md

```diff
@@ -0,0 +1,203 @@
+---
+authors: Doishy
+title: Alternative Pile Building
+---
+
+## Contents
+
+## Preamble
+
+In this chapter we will present some alternative piles which may be utilised
+depending on the tools available in your deck. This could include spicy main
+deck choices or the use of certain sideboard cards and is here to make you aware
+of how some of these tools are utilised in the wild. These are not listed in
+any specific order.
+
+## Shelldock Isle Piles
+
+You might see a lot of people running card:[SI] alongside card:[Em] in their 
+sideboards. The purpose of these cards is to use them in a pile that is 
+effective against blue based control decks which do not run cards like
+Wasteland. This pile often requires passing multiple turns and can be used
+in conjunction with Oracle. 
+
+> PTT
+
+:::row{variant=pile}
+- SI
+- Em
+- Island
+- CoS
+- TO
+:::
+
+In theory you can replace Emrakul with any card (including lands) however
+the uncounterable nature, the take an extra turn on casting (which still 
+works with Shelldock!) and the attack for 15 with Annihilator 6 all make it
+probably the most effective tool to use. 
+
+## Piles using Dragon's Rage Channeler
+
+Some people have tried various creature shells alongside the Doomsday package
+and one of the more interesting ones for use as part of the Doomsday
+resolution is card:[Dragon's Rage Channeler] (DRC). Each spell cast can let you "dig"
```

Comment:
```suggestion
resolution is card:[Dragon's Rage Channeler] (DRC). Each spell cast can let you *dig*
```

Also, feel free to add an entry here `src/tools/scryfall/cards.ts` as the card makes sense both for and against Doomsday as an archetype (and it's a pain to type, I often forget where the `'s` actually is)